server/tes.go

```diff
@@ -79,33 +85,61 @@ func (ts *TaskService) CancelTask(ctx context.Context, req *tes.CancelTaskReques
 	// updated database and other event streams
 	err = ts.Event.WriteEvent(ctx, events.NewState(req.Id, tes.Canceled))
 	if err == tes.ErrNotFound {
-		err = grpc.Errorf(codes.NotFound, fmt.Sprintf("%v: taskID: %s", err.Error(), req.Id))
+		err = status.Errorf(codes.NotFound, fmt.Sprintf("%v: taskID: %s", err.Error(), req.Id))
 	}
 	return &tes.CancelTaskResponse{}, err
 }
 
 // GetServiceInfo returns service metadata.
-func (ts *TaskService) GetServiceInfo(ctx context.Context, info *tes.ServiceInfoRequest) (*tes.ServiceInfo, error) {
+func (ts *TaskService) GetServiceInfo(ctx context.Context, info *tes.GetServiceInfoRequest) (*tes.ServiceInfo, error) {
 	resp := &tes.ServiceInfo{
-		Name: ts.Name,
-		Doc:  version.String(),
+		CreatedAt: "2016-03-21T16:27:49-07:00",
+		// TODO: Change this to "mailto:ellrott@ohsu.edu" when support for "mailto:" URL's are
+		// added to tes-compliance-suite
+		ContactUrl:       "https://ohsu-comp-bio.github.io/funnel/",
+		Description:      "Funnel is a toolkit for distributed task execution via a simple, standard API.",
+		DocumentationUrl: "https://ohsu-comp-bio.github.io/funnel/",
+		Environment:      "development",
+		Id:               "org.ga4gh.funnel",
+		Name:             ts.Name,
+		Organization: map[string]string{
+			"name": "OHSU Computational Biology",
+			"url":  "https://github.com/ohsu-comp-bio",
+		},
+		Storage: []string{
+			"file:///path/to/local/funnel-storage",
+			"s3://ohsu-compbio-funnel/storage",
+		},
```

Comment:
Seems like this would depend on the config. Google storage and ftp are also supported IIRC.